src/dns-test.js

```diff
@@ -293,19 +341,43 @@ async function sendQueries(nameservers_ipv4) {
  */
 function sendTelemetry(payload) {
     payload.measurementID = measurementID;
-    browser.telemetry.submitPing(TELEMETRY_TYPE, payload, TELEMETRY_OPTIONS);
+    console.log(payload);
+    // browser.telemetry.submitPing(TELEMETRY_TYPE, payload, TELEMETRY_OPTIONS);
+}
+
+async function fetchTest() {
+    // Unsure if we want to wrap the fetch in a try/catch. This would enable 
+    // us to handle network errors, in case that matters to us. I don't think 
+    // it does, though. I think all we want is to stop the measurement from 
+    // running if we don't get the response we expect. Also, if the fetch fails, 
+    // it doesn't make sense to send anything to telemetry, because that 
+    // may fail, too.
+    let response = await fetch("https://dnssec-experiment-moz.net/");
```

Comment:
`await fetch("...", {cache: "no-cache"});`?

see https://developer.mozilla.org/en-US/docs/Web/API/Request/cache

Reply:
Thanks for the pointer. I wonder if we should use `{cache: "reload"}` instead. This API call is supposed to act as an additional network connectivity test, but if we use `reload`, then we may also be able to determine whether there's network interference, which may be undesirable for our purposes.

Reply:
Added `{cache: "reload"}` for now in commit 5736eaa